host_core/native/hostcore_wasmcloud_native/src/lib.rs

```diff
@@ -72,7 +72,7 @@ fn get_oci_bytes(
 ) -> Result<Vec<u8>, Error> {
     task::TOKIO
         .block_on(async { oci::fetch_oci_bytes(&oci_ref, allow_latest, allowed_insecure).await })
-        .map_err(|_e| rustler::Error::Atom("Failed to fetch OCI bytes"))
+        .map_err(|_e| rustler::Error::Term(Box::new("Failed to fetch OCI bytes")))
```

Comment:
Why the need for this change? Granted,  my use of `Atom` is just from copying and pasting from the various examples available for rustler.

Reply:
So previously, it would return the value `:"Failed to fetch OCI bytes"`, which wouldn't trigger the match for `{:error, reason}` inside of the `start_executable_from_oci` function. It was technically possible to match against that atom, which I'm fine with doing, but changing it to Term (from the Error docs):
```
    /// Encodes an arbitrary Boxed Encoder and returns `{:error, term}` from
    /// the NIF. Very useful for returning descriptive, context-full errors.
```
So `Term` seemed to return the proper tuple there

Reply:
Hah.. super good catch. We should make sure that all the other Atoms in this project do the same thing, and return the Error term, e.g. `{:error, "string"}` .. which is what I thought `error::Atom` did based on the samples I saw.